### PR TITLE
Add Swedish translation for xdg-desktop-portal-cosmic to add Swedish …

### PR DESCRIPTION
**Pull request by bittin**
Add Swedish translation as i promised @f0x and @mmstick in the #development channel on the Mattermost

as i want COSMIC Screenshot to be in Swedish

### CODE REVIEW

**Comment by snaggen** [comment]
In the context of audio outputs, I think Utgång is the correct translation (unless we refer to the actual audio data stream, but that seems not to be the case here)

File: i18n/sv/xdg_desktop_portal_cosmic.ftl [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+capture = Fånga
+share = Dela
+save-to = Spara till
+    .clipboard = { save-to } Urklipp
+    .pictures = { save-to } Bilder
+    .documents = { save-to } Dokument
+choose-folder = Välj katalog
+
+share-screen = Dela din skärm
+    .description = Systemet vill dela innehållet på din skärm med "{$app_name}". Välj en skärm eller fönster att för att börja dela.
```
Comment (haaninjo) [inline i18n/sv/xdg_desktop_portal_cosmic.ftl:12]: One "att" too many. "Välj en skärm eller fönster för att börja dela"

Comment (bittin) [reply i18n/sv/xdg_desktop_portal_cosmic.ftl]: fixed

**Comment by bittin** [comment]
i think this can be merged now, if there is no other objections

**Review by jackpot51 — APPROVED** [review]

**Comment by bittin** [comment]
tested to install the AUR https://aur.archlinux.org/packages/cosmic-screenshot-git package that pulls from Github and this works 

@jackpot51 also sent PRs for updated Swedish for cosmic-files cosmic-launcher and cosmic-store earlier today

**Comment by bittin** [comment]
@haaninjo @snaggen @jackpot51 maybe this should be small letter cased too?, did not think about that

**Pull request merged**